app/src/main/java/io/gnosis/safe/ui/base/activity/BaseActivity.kt

```diff
@@ -33,6 +39,10 @@ abstract class BaseActivity : AppCompatActivity() {
         screenId()?.let {
             tracker.logScreen(it)
         }
+
+        lifecycleScope.launch {
+            notificationRepo.register()
+        }
```

Comment:
Should we be doing this on every activity that extends from `BaseActivity`? Maybe `SplashActiviy` as the launcher is the single place to do this?

Reply:
I am not sure how often we are required to run this method